packages/block/README.md

```diff
@@ -141,11 +141,11 @@ try {
 }
 ```
 
-EIP-1559 blocks have an extra `baseFeePerGas` field (default: `BigInt(7)`) and can encompass `FeeMarketEIP1559Transaction` txs (type `2`) (supported by `@ethereumjs/tx` `v3.2.0` or higher) as well as `LegacyTransaction` legacy txs (internal type `0`) and `AccessListEIP2930Transaction` txs (type `1`).
+EIP-1559 blocks have an extra `baseFeePerGas` field (default: `BigInt(7)`).
```

Comment:
I think we could remove this sentence, since `baseFeePerGas` is now a concept which I think is widely known (but we can keep this, no strong opinion).